domains/gbot.json

```diff
@@ -0,0 +1,13 @@
+{
+    "owner": {
+        "username": "Stef-00012",
+        "email": "admin@stefdp.is-a.dev",
+        "discord": "694986201739952229"
+    },
+    "record": {
+        "A": [
+            "46.248.190.217"
+        ],
+        "TXT": "cuttly-verification-site-2e4bc195cca78c21b0d87b6152303916ff415073-d2dd5123bf"
```

Comment:
We don't support TXT verification.

Reply:
removed the txt verification and changed the A record to my host (others with this record got merged so i guess this can be marked as resolved)

Reply:
deleted the file